### DISCUSSION TITLE
使用bytecodePlugin组件混淆代码后，打包出来的asar资源包，解包之后out\renderer\assets还是包含js源代码

### DISCUSSION THREAD

**Discussion by haolibote** [alex8088/electron-vite#566] [General]
![图片](https://github.com/user-attachments/assets/85537457-8b54-427f-a264-59c4e4a96cbb)
这个useCounterStore是在 src/main/stores/counter.js下的

**Comment by lling1105** [comment]
根据文档只对main和preload起作用
